zabbixDBAdapter.py

```diff
@@ -246,10 +250,14 @@ def signal_handler(signal, frame):
     # Listen for C-c and exit cleanly
     signal.signal(signal.SIGINT, signal_handler)
 
+    wavefront_sender = None
+    if SEND_TO_WF:
```

Comment:
Should we set this to True?

Reply:
As we have provisioned an option(SEND_TO_WF) either to print the output on the Terminal or send it to Wavefront, this parameter is being chosen by the end user.